src/tracing/cls.js

```diff
@@ -55,9 +54,9 @@ exports.startSpan = function startSpan(spanName, traceId, spanId) {
     span.p = parentSpan.s;
   // last resort, use newly generated Ids
   } else {
-    span.t = randomId;
+    span.t = tracingUtil.generateRandomTraceId();
   }
-  span.s = randomId;
+  span.s = tracingUtil.generateRandomSpanId();
```

Comment:
This will change the behavior of newly started root spans having the same TraceID and SpanID - is that ok?

Reply:
yes this is ok. We could also keep this, but that would make the code path messier.
root spans are identified by missing parent.